globals/Buffer.js

```diff
@@ -0,0 +1 @@
+export { Buffer } from 'node:buffer';
```

Comment:
I've added these files in a separate directory because this is the format that esbuild wants them to be in, so we don't want to run them through tsup.